node/src/components/consensus/tests.rs

```diff
@@ -1,4 +1,6 @@
-/// Module containing basic building blocks for the Discrete Event Simulator (DES).
+/// Basic building blocks for the Discrete Event Simulator (DES).
 pub(crate) mod consensus_des_testing;
-/// Module containing message queue.
+/// A mock consensus protocol for testing.
+pub(crate) mod mock_proto;
```

Comment:
Is it redundant to do `#[cfg(test)] pub(crate) mod mock_proto;`?

Reply:
I think so; would you prefer that, as a kind of documentation?
I don't think we usually do that, though, for subdirectories of `tests` directories; the other submodules in here don't have it either.

Reply:
Nah, if this isn't the convention let's just do what we normally do.